go.mod

```diff
@@ -5,4 +5,5 @@ go 1.16
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/stretchr/testify v1.3.0
+	golang.org/x/sys v0.27.0
```

Comment:
just to mention, this will not bring another dependency in kubernetes as it's already used there:
https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20golang.org%2Fx%2Fsys&type=code
and it's not an unwanted dependency.

https://github.com/kubernetes/kubernetes/blob/23e76432a9f2c602412d0d535fae4ac8d414d2ce/hack/unwanted-dependencies.json